frb_codegen/src/binary/commands.rs

```diff
@@ -172,6 +172,10 @@ pub(crate) struct GenerateCommandArgsPrimary {
     /// Dump all internal data. Same as `--dump` with all possible choices chosen.
     #[arg(long)]
     pub dump_all: bool,
+
+    /// List of cargo feature flags to enable when generating
+    #[arg(short = 'F', long)]
```

Comment:
nit: maybe we can allocate `short` flags later to avoid potential collision (the `long` may be enough for today)